src/accessories/characteristics/currentPosition.ts

```diff
@@ -16,58 +17,10 @@ export class CurrentPositionCharacteristic extends TuyaWebCharacteristic {
   }
 
   public getRemoteValue(callback: CharacteristicGetCallback): void {
-    this.accessory
-      .getDeviceState()
-      .then((data) => {
-        this.debug("[GET] %s", data?.state);
-        this.updateValue(data, callback);
-      })
-      .catch(this.accessory.handleError("GET", callback));
+    this.updateValue({}, callback);
```

Comment:
It looks like we are now no longer fetching the remote state 🤔 

What would happen if you use another system (outside of homekit) to update the state?
It seems like homekit wouldn't update with the new state.

Reply:
Yes, could be, but I'm not into the code. It is just the PR from bSr43@91ef52e you asked for.
Maybe you can combine the code? Would be awesome!
Is there a way we can donate for your work?